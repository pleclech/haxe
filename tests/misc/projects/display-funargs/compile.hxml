-main Main
--display Main.hx@61