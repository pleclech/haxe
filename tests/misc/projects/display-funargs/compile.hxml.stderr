<type opar='59' index='1'>
pos : Int -&gt; ?len : Null&lt;Int&gt; -&gt; String
</type>

